### PR TITLE
Update to the Pathfinder

### PR DESCRIPTION
**Pull request by Vectorial1024**
A number of updates.

- Expands selection to Tung Chung and LRT
- More lines
- Adjusted Materialize CSS parameters
- MOAR BADGES
- Badges now attempt to credit working tools featured in this Pathfinder
- Improved hit-count badge by wrapping the hit-count inside a proper Shields.io badge
- Gave name to this project: it is now called **探路 Pathfinder**
- Adjusted language: now uses "搜尋" instead of "查找"
- Might have renamed the var-names of some waypoints
- Fixed mysterious bug causing the newly-introduced "line name decoration" to fail (at least in my machine)
- Some LWB bus routes introduced; their bus routes use the html color "orange"
- Basically rewritten the README.md to better explain and coordinate possible collaboration

Still a lot can be done.

### CODE REVIEW

**Review by trafficguide — APPROVED** [review]
These are tested both on local machines and on the forked branch. Should be pretty stable.

**Comment by trafficguide** [comment]
As a matter of note, this pull request will be merged using the "Create merge commit" option instead of the previously-used "Squash and merge" option. It was found that doing "squash and merge" resulted in the forked branches accumulating more and more commits, making the UI in future pull requests increasingly cluttered. Using "create merge commit" should clear the clutter.

We will be using "create merge commit" from now on to prevent clutters. Please do not be alarmed when you see the commit count jump up by like 50+.

This merge has included a lot of new features, and most importantly, includes a rewritten README.md, clarifying possible roles that interested GitHub-ers may take in this project.

May glory be restored to Hong Kong.
